cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -257,9 +258,63 @@ class SpatialTest extends DocumentingTest {
           |<<cypher-spatial-crs, four coordinate reference systems>>.
           |This allows for both <<schema-index-use-a-single-property-index-with-where-using-equality, equality>>
           |and <<schema-index-use-index-with-where-using-range-comparisons, range>> queries using exactly the same syntax and behaviour as for other property types.
+          |If two range predicates are used, setting minimum and maximum points, this will effectively result in a
+          |<<schema-index-use-index-when-executing-a-spatial-bounding-box-search, 'bounding box query'>>.
           |In addition, queries using the `distance` function can, under the right conditions, also use the index, as described in the section
-          |<<schema-index-use-index-when-executing-a-spatial-distance-search, 'Use index when executing a spatial distance search'>>
+          |<<schema-index-use-index-when-executing-a-spatial-distance-search, 'Use index when executing a spatial distance search'>>.
         """.stripMargin)
     }
+    section("Comparability and Orderability", "cypher-comparability-orderability") {
+      p(
+        """
+          |Points with different CRS are not comparable. This means that any function operating on two points of different type will return null.
+          |This is true of the distance function as well as inequality comparisons. If these are used in predicate, they will cause the associate match
```

Comment:
More line splitting

Reply:
I would hyperlink to "the distance function"

Reply:
"If these are used in predicate" -> ??? "If these are used as predicates" (or did you mean something else?)
"they will cause the associate match" -> "they will cause the associated match"